neuracore/ml/utils/policy_inference.py

```diff
@@ -0,0 +1,396 @@
+"""TODO."""
+
+import logging
+import tempfile
+from pathlib import Path
+from typing import Optional
+
+import numpy as np
+import requests
+import torch
+import torchvision.transforms as T
+from PIL import Image
+
+from neuracore.core.auth import get_auth
+from neuracore.core.const import API_URL
+from neuracore.core.nc_types import (
+    CameraData,
+    CustomData,
+    EndEffectorData,
+    JointData,
+    LanguageData,
+    ModelPrediction,
+    PointCloudData,
+    PoseData,
+    SyncPoint,
+)
+from neuracore.core.utils.download import download_with_progress
+from neuracore.ml import BatchedInferenceSamples, MaskableData
+from neuracore.ml.utils.nc_archive import load_model_from_nc_archive
+
+logger = logging.getLogger(__name__)
+
+
+class PolicyInference:
+    """Todo."""
```

Comment:
Please improve the description
```suggestion
    """This class restructures the data from a sync point to run infrence over a given model"""
```